kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -24,10 +24,17 @@ const publicMethods = [
  * is configured in webpack.config.js
  * @classdesc
  */
-function Lib() {
+function Lib(kolibriGlobal) {
   // libraries
   this.loglevel = require('loglevel');
   this.vue = require('vue');
+  this.vue.mixin({
+    methods: {
+      url: (url_name, ...args) => {
+        return kolibriGlobal.urls[url_name](...args);
+      }
```

Comment:
What are the scenarios in which we use this reverse functionality? Can you give an example of it in use? It looks like `kolibriGlobal.urls` is an object of functions but it's not clear to me what parameters those functions take.

I agree with @MCGallaspy that it's easy to just reference `kolibriGlobal.urls` directly, rather than going through an extra level of indirection.